service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -1,18 +1,42 @@
 #!/bin/bash
 #
-# Default post startup script for GCP notebooks.
-# GCP Notebook post startup scrips are run only when the instance is first created.
+# Name: post-startup.sh
+#
+# Description
+#   Default post startup script for Google Cloud Vertex AI Workbench VM
+#   running JupyterLab.
+#
+# Exection details
+#   The post-startup script runs on Vertex AI notebook VMs during *instance creation*;
+#   it is not run on every instance start.
+#
+#   *** The post-startup script runs as root. ***
+#
+#   The startup script is executed from /opt/c2d/scripts/97-run-post-startup-script.sh
```

Comment:
Is this information documented externally? Would be great if you could add a link here.